Lib/test/test_pyrepl/test_pyrepl.py

```diff
@@ -1757,3 +1757,14 @@ def test_showrefcount(self):
         output, _ = self.run_repl("1\n1+2\nexit()\n", cmdline_args=['-Xshowrefcount'], env=env)
         matches = re.findall(r'\[-?\d+ refs, \d+ blocks\]', output)
         self.assertEqual(len(matches), 3)
+
+    def test_detect_pip_usage_in_repl(self):
+        for pip_cmd in ("pip", "pip3", "python -m pip", "python3 -m pip"):
+            with self.subTest(pip_cmd=pip_cmd):
+                output, exit_code = self.run_repl([f"{pip_cmd} install antigravity", "exit"])
```

Comment:
I know this doesn't actually get called, but let's use a package owned by the [PyPA](https://pypi.org/project/sampleproject/) rather than a [third-party](https://pypi.org/project/antigravity/):
```suggestion
                output, exit_code = self.run_repl([f"{pip_cmd} install sampleproject", "exit"])
```